File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        for i in range(len(nums)):
            new_list = nums[:i] + nums[i+1:]
            for n in new_list:
                if nums[i] == n:
                    return True
        return False

                
        

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer, Integer> map = new HashMap<>();
        int[] returned = new int[k];
        for (int i : nums) {
            if (!map.containsKey(i)) {
                map.put(i, 0);
            }
            map.put(i, map.get(i) + 1);
        }
        PriorityQueue<Integer> pq = new PriorityQueue<>(new Comparator<Integer>(){
            @Override
            public int compare(Integer a, Integer b) {
                return map.get(b).compareTo(map.get(a));
            }
        });
        for (int key : map.keySet()) {
            pq.add(key);
        }
        for (int i = 0; i < k; i++) {
            returned[i] = pq.poll();
        }
        return returned;
    }
}
